po/Makefile.am

```diff
@@ -90,7 +90,8 @@ po/cockpit.js.pot: po/POTFILES.js.in
 		--keyword=gettext:1,1t --keyword=gettext:1c,2,2t \
 		--keyword=ngettext:1,2,3t --keyword=ngettext:1c,2,3,4t \
 		--keyword=gettextCatalog.getString:1,3c --keyword=gettextCatalog.getPlural:2,3,4c \
-		--from-code=UTF-8 --directory=$(srcdir) --files-from=$^ | \
+		--from-code=UTF-8 --directory=$(srcdir) --files-from=$^ \
+		--add-comments=Translators | \
```

Comment:
How did this come to life? Is it needed? If is, it definitely definitely be a different commit.

Reply:
I was doing and redoing the translatable strings and some required translator comments. Without this, they aren’t extracted, but I guess I could split it off.

Reply:
ah cool. Please split it to different commit or even maybe a different PR (ideally with demoing on some string and documenting it (ideally in https://github.com/cockpit-project/cockpit/tree/master/pkg/playground in the translate subsection)

Reply:
Made the strings simpler, this is no longer there.